testing/integration/postman/BioHubBC-Environment-DEV.postman_environment.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
We can just revert this file entirely, as we never use the object store env vars anywhere in postman.